xml/src/main/java/org/ehcache/xml/CacheManagerServiceConfigurationParser.java

```diff
@@ -34,4 +34,8 @@
   URI getNamespace();
 
   ServiceCreationConfiguration<T> parseServiceCreationConfiguration(Element fragment);
+
+  Class<T> handleServiceCreationConfigurationType();
+
+  void translateServiceConfiguration(Document doc, ServiceCreationConfiguration<T> serviceCreationConfiguration);
```

Comment:
unmarshal?

Reply:
Huh? In this context would it be:
`marshal == unparse`
`parse == unmarshal`

Reply:
I'm slightly lost but
* From xml to bean = unmarshal
* From bean to xml = marshal

Are we in sync?

Reply:
Yes I think we are... but just to confirm if you wanted to switch to the marshal terminology:
```java
Document marshalServiceCreationConfiguration(ServiceCreationConfiguration<T> config);
ServiceCreationConfiguration<T> unmarshalServiceCreationConfiguration(Document config);
```

Reply:
So I will alter the api and new api will look like:

void marshalServiceConfiguration(Document doc, ServiceCreationConfiguration<T> serviceCreationConfiguration); 

Let me know if it is otherwise